templates/sso74-postgresql-persistent.json

```diff
@@ -5,19 +5,19 @@
         "annotations": {
             "iconClass" : "icon-sso",
             "tags" : "sso,keycloak,jboss,hidden",
-            "version": "7",
-            "openshift.io/display-name": "Red Hat Single Sign-On CD 7 TechPreview + PostgreSQL (Persistent with passthrough TLS)",
+            "version": "7.4.0.GA",
```

Comment:
prefix with `v` (both for `version` & `rhsso` value below)

Reply:
@iankko What is the justification for this suggestion?

Reply:
Same as above. The `v` form is used in the instructions on how to install the updated templates

Reply:
Which instructions? Should they be then updated?
7.3 templates never used `v` form so I'm now confused why we should introduce it now.

Reply:
Ah, right. You are correct. We use the `v` prefix only for tags. Ignore my previous comments